LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/CategoryController.java

```diff
@@ -0,0 +1,24 @@
+package com.target.ready.library.system.service.LibrarySystemService.Controller;
+
+import com.target.ready.library.system.service.LibrarySystemService.Entity.Category;
+import com.target.ready.library.system.service.LibrarySystemService.Service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("library_service_api/v2")
```

Comment:
"library/v2" how about this? Everyone know its an api and its a service.

Reply:
Sure